Helper/QueryFilterHelper.php

```diff
@@ -160,8 +163,20 @@ private function getCustomValueValueExpression(
         SegmentQueryBuilder $customQuery,
         string $tableAlias,
         string $operator,
-        string $valueParameter
+        string $valueParameter,
+        bool $alreadyNegated = false
     ) {
+        if ($alreadyNegated) {
+            switch ($operator) {
+                case 'empty':
```

Comment:
Is it not better to use OperatorOptions::EMPTY instead of directly using the sign? However, as I noticed in this file, we just use the direct sign only. Was there any specific reason?

Reply:
I have no idea. I just followed the values used within the existing method.

Reply:
ok